F/FastJet_Julia_Wrapper/build_tarballs.jl

```diff
@@ -3,10 +3,9 @@
 using BinaryBuilder
 using Pkg
 
-julia_version = v"1.5.3"
-
 name = "FastJet_Julia_Wrapper"
-version = v"0.8.6"
+version = v"0.9.0"
```

Comment:
The source code did not change; do you really want to bump the version to indicate breaking changes? There seems to be no need for that?